js/templates/modals/connectionManagement/configurations.html

```diff
@@ -1,14 +1,14 @@
-<div class="contentBox padMd clrP clrBr clrSh3">
+<div class="contentBox padMd clrP clrBr clrSh3" style="min-height: 500px">
```

Comment:
It would be better to give this a one-off class with the minimum height, and put it in _connectionManagement.scss.